mobile.js

```diff
@@ -1,17 +1,95 @@
+  linkdemo: 'https://maedea9.github.io/portfolio1/',
+  github: 'https://github.com/Maedea9/portfolio1',
+},
+{
+  image: 'images/Rectangle21v2.png',
+  name: 'Project name',
+  languages: ['Html/Css', 'Ruby on Rails', 'JavaScript'],
+  description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+  linkdemo: 'https://maedea9.github.io/portfolio1/',
+  github: 'https://github.com/Maedea9/portfolio1',
+},
+{
+  image: 'images/Rectangle21(1).png',
+  name: 'Project name',
+  languages: ['Html/Css', 'Ruby on Rails', 'JavaScript'],
+  description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+  linkdemo: 'https://maedea9.github.io/portfolio1/',
+  github: 'https://github.com/Maedea9/portfolio1',
+},
+{
+  image: 'images/Rectangle21(2).png',
+  name: 'Project name',
+  languages: ['Html/Css', 'Ruby on Rails', 'JavaScript'],
+  description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+  linkdemo: 'https://maedea9.github.io/portfolio1/',
+  github: 'https://github.com/Maedea9/portfolio1',
+},
+{
+  image: 'images/Rectangle21(3).png',
+  name: 'Project name',
+  languages: ['Html/Css', 'Ruby on Rails', 'JavaScript'],
+  description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+  linkdemo: 'https://maedea9.github.io/portfolio1/',
+  github: 'https://github.com/Maedea9/portfolio1',
+},
+];
 
-const hamburguerIcon = document.querySelector('.fa-sharp');
-const closeIcon = document.querySelector('.fa-xmark');
+function createmodal(n) {
```

Comment:
[OPTIONAL] Well done overall 💯 but how about you use more descriptive variable names 🤔 : Use variable names that are more descriptive and meaningful, which makes the code easier to read and understand. For example, instead of using `n` as a variable name, use `index`. 🤓

Reply:
Thank you @didierganthier will certainly make those changes! Sorry about the branch name I forgot to create one and reused the old one. Thanks again you're the best!